charts/sn-platform-slim/values.yaml

```diff
@@ -1407,10 +1407,15 @@ pulsar_detector:
   extraVolumeMounts: []
   extraEnv: []
   ## Additional pulsar_detector container arguments
-  extraArgs: {}
+  extraArgs:
```

Comment:
Since all the values are commented out, do you want to keep it in this format `extraArgs: {}`?